source/index.pug

```diff
@@ -21,10 +21,10 @@ ul#intro-feature-list
       .intro-feature-icon
         i.fa.fa-cloud-upload
       h3.intro-feature-title One-Command Deployment
-      p.intro-feature-desc You only need one command to deploy your site to GitHub Pages, Heroku or other platforms.
+      p.intro-feature-desc You only need one command to deploy your site to GitHub Pages, Firebase Hosting, Heroku, Netlify, ZEIT or other platforms.
   li.intro-feature-wrap
     .intro-feature
       .intro-feature-icon
         i.fa.fa-cog
       h3.intro-feature-title Plugins
-      p.intro-feature-desc Features powerful APIs for limitless extensibility. Various plugins are available to support most template engines (EJS, Pug, Nunjucks, and many others). Easily integrate with existing NPM packages (Babel, PostCSS, Less/Sass, etc).
\ No newline at end of file
+      p.intro-feature-desc Various plugins are available to generate sitemap and RSS, insert player for music or video, intergrate search engine and support most template engines.
```

Comment:
"Various plugins are available to further enhance your site. Easily add RSS feed, embed audio/video, search engine integration/optimization, add support for most template engines and many more."

---

I use "audio" instead of "music" to cover podcast and radio.